ethereum/core/src/test-support/java/tech/pegasys/pantheon/ethereum/core/InMemoryStorageProvider.java

```diff
@@ -57,6 +59,16 @@ public WorldStateStorage createWorldStateStorage() {
     return new KeyValueStorageWorldStateStorage(new InMemoryKeyValueStorage());
   }
 
+  @Override
+  public PrivateTransactionStorage createPrivateTransactionStorage() {
+    return null;
```

Comment:
Returning null here might lead to errors in the future. Wouldn't it be better to return a stub implementation of the `PrivateTransactionStorage`?